### PR TITLE
[NCL-6006] When milestone close finishes, close result page doesn't r…

### PR DESCRIPTION
**Pull request by jomrazek**
…efresh with result.

### CODE REVIEW

File: ui/app/common/pnc-client/message-bus/listeners/milestonePushListener.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+/*
```
Comment (jomrazek) [inline ui/app/common/pnc-client/message-bus/listeners/milestonePushListener.js:1]: added same type listener and notifications as existing for brew push, we're not using them, don't know if that's bug or intention to use them in different way in future

**Comment by jomrazek** [comment]
https://github.com/project-ncl/pnc-js-libs/pull/12

File: ui/app/product-milestones/close-result/pncProductMilestoneCloseResultPage.js [hunk 2]
```diff
@@ -43,12 +43,23 @@
...
     };
 
     function showTable() {
         return $ctrl.data.buildPushResults && $ctrl.data.buildPushResults.length;
     }
 
+    function load(milestonePushResult) {
+      if (milestonePushResult) {
+        $ctrl.data = milestonePushResult;
+        $ctrl.prefixFilters = 'loggerName.keyword:org.jboss.pnc.causeway|org.jboss.pnc._userlog_,level.keyword:INFO|ERROR|WARN';
```
Comment (dominikbrazdil) [inline ui/app/product-milestones/close-result/pncProductMilestoneCloseResultPage.js:59]: Is this changing? It could be omitted if not

**Review by dominikbrazdil — APPROVED** [review]

File: ui/app/product-milestones/close-result/pncProductMilestoneCloseResultPage.js (hunk 2 above)
Comment (jomrazek) [reply ui/app/product-milestones/close-result/pncProductMilestoneCloseResultPage.js]: yes, you're right, thanks!

**Review by DnsZhou — APPROVED** [review]
For the front-end part of the code, it looks good to me. Although the trigger for WS Message seem not activated because of no WS message from the backend when a milestone is closed, it might more related to the backend WS issue instead of the frontend part.

**Comment by DnsZhou** [comment]
Verified with stage environment.

**Pull request merged**